shared/git/row/index.js

```diff
@@ -185,6 +195,43 @@ class Row extends React.Component<Props, State> {
                 <Text type="BodySmall">.</Text>
               </Text>
             </Box>
+            {!!this.props.teamname && (
+              <Box style={{...globalStyles.flexBoxRow, alignItems: 'center', marginTop: globalMargins.xtiny}}>
+                {this.props.canEdit && (
+                  <Checkbox
+                    checked={!this.props.chatDisabled}
+                    onCheck={this.props.onToggleChatEnabled}
+                    label=""
+                    labelComponent={
+                      <Text type="BodySmall">
+                        Announce pushes in{' '}
+                        <Text
+                          type={this.props.chatDisabled ? 'BodySmall' : 'BodySmallPrimaryLink'}
+                          onClick={e => {
+                            if (this.props.chatDisabled) {
+                              return
+                            }
+                            e.preventDefault()
+                            this.props.onOpenChannelSelection()
+                          }}
+                        >
+                          {this._channelNameToString(this.props.channelName)}
```

Comment:
there's a bug on mobile where this channel text doesn't update after setting a new channel (it is set though). I didn't spend much time on it because mobile git management probably isn't common